common/util/rviz_plugins/autoware_vehicle_rviz_plugin/README.md

```diff
@@ -0,0 +1,59 @@
+# autoware_vehicle_rviz_plugin
+
+This package is including jsk code.  
+Note that jsk_overlay_utils.cpp and jsk_overlay_utils.hpp are BSD license.
+
+## Purpose
+
+This plugin provides a visual and easy-to-understand display of vehicle speed, turn signal and steering status.
+
+## Inputs / Outputs
+
+### Input
+
+| Name                       | Type                                                    | Description                        |
+| -------------------------- | ------------------------------------------------------- | ---------------------------------- |
+| `/vehicle/status/twist`    | `geometry_msgs::msg::TwistStamped`                      | The topic is vehicle twist         |
+| `/control/turn_signal_cmd` | `autoware_auto_vehicle_msgs::msg::SteeringReport`       | The topic is status of turn signal |
+| `/vehicle/status/steering` | `autoware_auto_vehicle_msgs::msg::TurnIndicatorsReport` | The topic is status of steering    |
+
+## Parameter
+
+### Core Parameters
+
+| Name                            | Type   | Default Value        | Description                              |
+| ------------------------------- | ------ | -------------------- | ---------------------------------------- |
+| `property_text_color_`          | QColor | QColor(25, 255, 240) | Text color                               |
+| `property_left_`                | int    | 128                  | Left of the plotter window [px]          |
+| `property_top_`                 | int    | 128                  | Top of the plotter window [px]           |
+| `property_width_`               | int    | 128                  | Left of the plotter window [px]          |
+| `property_height_`              | int    | 128                  | Width of the plotter window [px]         |
+| `property_length_`              | int    | 256                  | Height of the plotter window [px]        |
+| `property_value_height_offset_` | int    | 0                    | Height offset of the plotter window [px] |
+| `property_value_scale_`         | float  | 1.0 / 6.667          | Value scale                              |
+| `property_handle_angle_scale_`  | float  | 3.0                  | Scale is steering angle to handle angle  |
+| `velocity_`                     | float  | 0.0                  | Velocity [m/s]                           |
```

Comment:
This parameter seems not to exist.

Reply:
fix https://github.com/tier4/AutowareArchitectureProposal.iv/pull/591/commits/fc7d62b66aa0af5dcc3f45065806684a1e900ada